framework/include/base/MooseApp.h

```diff
@@ -570,12 +570,6 @@ class MooseApp : public ConsoleStreamInterface, public libMesh::ParallelObject
   /// Populate the _syntax object with mesh only tasks that will be executed before writing mesh.
   virtual void modifyMeshOnlyTasks(Syntax &) {}
```

Comment:
@rwcarlsen - This is the new API that I just added. It _IS_ being overridden in Rattlesnake so after this gets merged in, we'll need to pull it out of Rattlesnake and then pull it out of MOOSE. Let's not leave it hanging around because it's a dead function now.